assets/subworkflows/espresso/processing_find_minima.yml

```diff
@@ -0,0 +1,23 @@
+application:
+    name: python
+    version: 3.8.6
+method:
+    name: UnknownMethod
+model:
+    name: UnknownModel
+name: Find ESP Value
```

Comment:
We should either remove the "specifics", such as mention of ESP, call AVG_ESP -> "ARRAY_TO_FIND_MINIMA_IN", or smth like that, and keep this as a python flavor instead of espresso. Or, alternatively, call this `average_electrostastatic_potential_find_minima`

Reply:
Unfortunately, I cannot use this subworkflow as a python asset without major refactoring. Right now the workflow creation occurs via application, so I can't mix `espresso` and `python` assets. I believe for the same reason, `espresso_xml_get_qpt_irr.yml`, `pre_processor.yml`, and `post_processor.yml` were put in this directory instead of their corresponding application folder.
Nevertheless, I think the use of "specifics" in this subworkflow is legitimate, so I would prefer the second option, i.e. renaming the file.